src/operators/frei0r_op.ml

```diff
@@ -23,6 +23,10 @@
 open Source
 open Extralib
 
+let () =
+  Lang.add_module "frei0r";
+  Lang.add_module "video.frei0r"
```

Comment:
Both?

Reply:
Fixed in b11641b8730a0d6bff639dcb7362aeefb7e0168e.